pangea-sdk/service/ai_guard/api.go

```diff
@@ -26,6 +26,11 @@ func (e *aiGuard) GuardText(ctx context.Context, input *TextGuardRequest) (*pang
 	return request.DoPost(ctx, e.Client, "v1/text/guard", input, &TextGuardResult{})
 }
 
+type TopicDetectionOverride struct {
+	Disabled  *bool     `json:"disabled,omitempty"`
+	BlockList *[]string `json:"block_list,omitempty"`
```

Comment:
I think we tried to avoid pointers to slices in the past. A slice could directly be nil too.

Reply:
Ah. Removed.